### ISSUE TITLE
Add support for get-in/2

### ISSUE THREAD

**Issue by oubiwann** [lfex/lutil#32]
A la Clojure's [get-in](https://clojuredocs.org/clojure.core/get-in) function.

Support the following types by atom keys
 * proplists
 * dicts
 * orddicts

Support the following types by integer indices:
 * nested lists (using ``nth``)
 * proplists (using ``element/2``)

This blocks the following:
 * https://github.com/lfex/lcfg/issues/3
 * https://github.com/lfex/lcfg/issues/2

**Comment by oubiwann** [comment]
Lovely: for nested lists, it's just this simple:

```cl
(lists:foldl #'lists:nth/2 data keys))
```

**Comment by oubiwann** [comment]
And for nested proplists, just as beautiful:

```cl
(lists:foldl #'proplists:get_value/2 data keys)
```

**Comment by oubiwann** [comment]
Released as part of 0.6.1.
